ModuleData/tow_abilitytemplates.xml

```diff
@@ -321,7 +321,7 @@
        StringID="ShadowStep"
        Name="Shadow Step"
        SpriteName="windofdeath_icon"
-       CoolDown="100"
+       CoolDown="5"
```

Comment:
i sense some fireball machine gun  vibes :)

Reply:
CoolDown isnt really relevant on SpecialMove anymore since it has the charged mechanic. It doesnt matter if the cooldown has expired, you cant activate it again until its charged.